src/citrine/informatics/descriptors.py

```diff
@@ -252,3 +253,11 @@ def __str__(self):
 
     def __repr__(self):
         return "FormulationDescriptor(key={})".format(self.key)
+
+    @staticmethod
+    def hierarchical() -> "FormulationDescriptor":
```

Comment:
I'm not sure what we want to call this for the SDK?

Reply:
@jspeerless thoughts on user-facing language?

Reply:
James said he likes "hierarchical" here